frontend/javascripts/dashboard/dataset/dataset_settings_data_tab.tsx

```diff
@@ -529,6 +529,17 @@ function SimpleLayerForm({
                               ),
                             ),
                     },
+                    {
+                      warningOnly: true,
+                      validator: (_rule, value) =>
+                        value != null && value === 2 ** bitDepth - 1
+                          ? Promise.reject(
```

Comment:
Why is this an error? Maybe the segmentation has used up all available ids? Ideally, it should be a warning, I think.

Reply:
This is not an error. The field `warningOnly` configures the validator to show the rejected message as a warning.
![image](https://github.com/scalableminds/webknossos/assets/39529669/2d67e193-c485-47c5-968e-81d1107c47c4)

Reply:
Ah, great, nevermind then :)